config/initializers/statsd.rb

```diff
@@ -61,6 +65,10 @@ class << Samson
     "stage:#{payload.fetch(:stage)}",
   ]
 
+  # report if things were run with kubernetes
+  kubernetes = payload.fetch(:kubernetes)
+  tags << "kubernetes:#{kubernetes}" unless kubernetes.nil?
```

Comment:
```suggestion
  tags << "kubernetes:#{!!payload.fetch(:kubernetes)}"
```

Reply:
want to keep it consistent with the above tag, but also not send any kubernetes-related tag if it's for a build (which is when `kubernetes` would be `nil`)